src/components/interfaces/HMI_API.xml

```diff
@@ -2509,14 +2509,26 @@
   <param name="compassDirection" type="Common.CompassDirection" mandatory="false">
     <description>See CompassDirection.</description>
   </param>
-  <param name="pdop" type="Float" minvalue="0" maxvalue="10" mandatory="false">
+  <param name="pdop" type="Float" minvalue="0" maxvalue="1000" mandatory="false" since="5.0">
     <description>PDOP.</description>
+    <history>
+      <param name="pdop" type="Float" minvalue="0" maxvalue="10" defvalue="0" mandatory="true" until="5.0">
```

Comment:
somehow mandatory is no longer "true" in develop branch, so this history need to a update to reflect that.

Reply:
@yang1070 Thank you. Fixed in 184ef635260373b1cb7b193010c77e18f7c0f8cc